src/jmcomic/jm_option.py

```
@@ -328,6 +330,55 @@ def compatible_with_old_versions(cls, dic):
+                    jm_log('option.migrate',
+                           f'[zip 插件迁移] level 参数已过时，建议删除level参数。'
+                           f'你的当前配置为，在本子下载完毕后按章节压缩，建议改为如下的等价新写法：\n'
+                           f'plugins:\n'
+                           f'  after_photo:\n'
+                           f'    - plugin: {pinfo["plugin"]}\n'
+                           f'      kwargs: {pinfo["kwargs"]}\n'
+                           )
+                else:
+                    if level != 'photo':
+                        jm_log('option.migrate',
+                               f'[zip 插件迁移] level 参数已过时，你可以直接删除该参数，不会有任何影响')
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)